client/lib/media/test/actions.js

```diff
@@ -69,6 +69,7 @@ describe( 'MediaActions', function() {
 						return {
 							get: mediaGet.bind( [ siteId, mediaId ].join() ),
 							update: mediaUpdate.bind( [ siteId, mediaId ].join() ),
+							edit: mediaUpdate.bind( [ siteId, mediaId ].join() ),
```

Comment:
Not too much. In fact, both methods `update()` and `edit()` are very similar because, in fact, the endpoints are very similar as well. :-o